scripts/zones/Bibiki_Bay/mobs/Shen.lua

```diff
@@ -2,14 +2,63 @@
 -- Area: Bibiki Bay
 --  Mob: Shen
 -----------------------------------
+local ID = require("scripts/zones/Bibiki_Bay/IDs")
+-----------------------------------
 local entity = {}
 
+
+entity.enterShell = function(mob)
+    mob:setAnimationSub(1)
+    mob:SetAutoAttackEnabled(false)
+    mob:SetMagicCastingEnabled(false)
+    mob:setMod(xi.mod.REGEN,        120)
+    mob:setMod(xi.mod.UDMGPHYS,   -9000)
+    mob:setMod(xi.mod.UDMGRANGE,  -9000)
+    mob:setMod(xi.mod.UDMGMAGIC,  -6000)
+    mob:setMod(xi.mod.UDMGBREATH, -6000)
+    mob:setMobMod(xi.mobMod.NO_MOVE,  1)
+end
+
+entity.exitShell = function(mob)
```

Comment:
both enterShell and existShell should be local functions, IMO

Reply:
we actually have a mixin with weaker values, maybe the potency in the mixin should be configurable per mob so that pr authors for NMs don't have to duplicate code

Reply:
oh, true, if a mixing exists, that should be used

Reply:
+1 for reuse the existing mixin